File: javascript/exercicio5.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function inverterString(string) {
    let stringInvertida = '';
    for (let i = string.length - 1; i >= 0; i--) {
        stringInvertida += string[i];
    }
    return stringInvertida;
}

rl.question("Digite uma string: ", (entrada) => {
    const stringInvertida = inverterString(entrada);
    console.log("String invertida:", stringInvertida);
    rl.close();
});

File: python/exercicio5.py
def inverter_string(s):
    inverted = ''
    for i in range(len(s) - 1, -1, -1):
        inverted += s[i]
    return inverted

string = input("Digite uma string: ")
inverted_string = inverter_string(string)
print("String invertida:", inverted_string)
